src/data/local_storage.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * @module Data
+ * @submodule LocalStorage
+ * @requires core
+ *
+ * This module defines the p5 methods for working with local storage
+ */
+
+'use strict';
+var p5 = require('../core/main');
+/**
+ *
+ * Stores a value in local storage under the key name. The key can
```

Comment:
can you add one more sentence here explaining what local storage is? for example:
"Local storage is saved in the browser and persists between browsing sessions and page reloads."
I wonder if we also should include some kind of security warning. Local storage is pretty insecure so users should not store sensitive info there. Or maybe there is a link we can link to with more info about local storage?

Reply:
Good idea!

 I couldn't find a good link for security stuff. I found a number of pages where folks talk about how insecure local storage is but everything I found was not necessarily appropriate for beginners and/or had information that would be confusing relative to this API for working with it (such as statements like 'localStorage is terrible because it can only store strings despite being a giant JavaScript object'). I don't want to accidentally lead folks down a rabbit-hole of suggestions about cryptographic-signed cookies.

I think your suggestion plus a simple statement about sensitive information should suffice.
"Local storage is saved in the browser and persists between browsing sessions and page reloads. Sensitive data such as passwords or personal information should not be stored in local storage."

Reply:
I think that sounds great